README.md

```diff
@@ -77,4 +77,41 @@ from GitHub Actions, specifying the desired new version in the "Keycloak version
 The job can also be triggered manually using GitHub client:
 ```shell
 gh workflow run "Keycloak AMD and ARM" --ref <git_branch> -f keycloak_version=<desired_keycloak_version>
-```
\ No newline at end of file
+```
+
+## Branch deployments
+
+If for some reason the keycloak config volume gets deleted, we will need to run an init container which copies the 
+keycloak config into the volume.
+
+The following code will need adding inside the spec section of `deployment.yaml`: 
+
+```yaml
+      initContainers:
+        - name: keycloak-persistent-volume
+          image: {{ .Values.image.repo }}keycloak:{{ .Values.keycloakVersion }}
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: keycloak-config-volume
+              mountPath: /keycloakVolume
+          command: ["/bin/bash", "-c"]
+          args:
+            - cp -r opt/jboss/. keycloakVolume
+```
+
+It will only need to be run once and the code can be removed after the init container has run. The reason for doing this
```

Comment:
Can we make this go away as an issue by changing the deployment spec
.spec.strategy.type to recreate